__tests__/app.test.js

```diff
@@ -124,115 +124,167 @@ describe("GET /api/reviews/:review_id", () => {
     });
 })
 
-
-
-describe("PATCH /api/reviews/:review_id", () => {
-    test("status code 200, responds with a changed vote count (increase)", () => {
-        const newVote = { inc_votes: 20 };
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("status code 200, responds with the comments for the corresponding review", () => {
+        const review_id = 3;
         return request(app)
-            .patch('/api/reviews/2')
-            .send(newVote)
+            .get(`/api/reviews/${review_id}/comments`)
             .expect(200)
             .then(({ body }) => {
-                expect(body.review).toEqual({
-                    review_id: 2,
-                    title: 'Jenga',
-                    designer: 'Leslie Scott',
-                    owner: 'philippaclaire9',
-                    review_img_url:
-                        'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
-                    review_body: 'Fiddly fun for all the family',
-                    category: 'dexterity',
-                    created_at: new Date(1610964101251).toISOString(),
-                    votes: 25
+                const { comments } = body;
+                expect(!!comments.length).toEqual(true)
```

Comment:
Is this just to be extra expressive !!??

Reply:
no, it's the poor man's type cast 🤣